spec.md

```diff
@@ -190,6 +190,9 @@ If present, the value of this header MUST be a digest matching that of the respo
 
 If the blob is not found in the registry, the response code MUST be `404 Not Found`.
 
+A GET request MAY also include a `Range` request header to download part of a
+blob in accordance with [RFC 9110](https://datatracker.ietf.org/doc/html/rfc9110#name-range-requests).
```

Comment:
I reworded it to focus on the registry since "a request MAY include" could imply "a registry MUST support" which I don't think is the goal.

Also a couple nits:

- One sentence per line for easier PRs (remove the linefeed here).
- I think we should standardize on a single RFC host. It looks like most of the links here use https://www.rfc-editor.org/.

```suggestion
A registry SHOULD support the `Range` request header in accordance with [RFC 9110](https://www.rfc-editor.org/rfc/rfc9110.html#name-range-requests).
```